apps/govquests-api/rails_app/app/graphql/mutations/start_action_execution.rb

```diff
@@ -39,6 +39,10 @@ def resolve(quest_id:, action_id:, action_type:, send_email_verification_input:
         {
           address: current_user.address
         }
+      when "governance_voter_participation"
+        {
+          address: current_user.address
+        }
```

Comment:
I forgot to merge PR #78  before, it might have some conflicts with this file now